css/style.css

```diff
@@ -0,0 +1,40 @@
+body{
+    background: #7373BC;
+    color: white;
+    text-align: left;
+}
+
+#main-title{
+    text-align: center;
+}
+
+#description {
+    font-weight: bold;
+}
+
+#survey-form{
+    display: flex;
+    flex-direction: column;
+    background: #2B2B49;
+    margin-left: 25%;
+    margin-right: 25%;
+    padding: 2%;
+}
+
+@media (max-width: 768px){
+    #survey-form{
+        margin-left: 5%;
+        margin-right: 5%;
+    }
+}
+
+#submit {
+    width: 10em;
+    height: 2em;
+    align-self: center;
+    font-size: 1.1em;
+    margin-top: 5%;
+}
```

Comment:
Try to use "px" unit or "rem" unit, because "em" is don't usually used.

Reply:
- [ ] Please make the comments polite and easy to understand. Using a polite tone makes communication smoother while easy-to-understand comments would help the author understand what changes need to be made. 😄📗 

 #### Example of Polite comments 
 
> **_Kindly_** use `rem` or `em` units instead of `px`. 
> I think it would be better to use `rem` or `em` units instead of `px`. 

#### Example of a clear and easy-to-understand comment

 > **_Kindly_** use `rem` or `em` units instead of `px`. Using `rem` or `em` units meets different web accessibility standards and scales better in different media queries. 👍 😎 


 
 #### Hint: You don't have to create another inline comment. You could just edit the inline comments that you have above. 💻